website/versioned_docs/version-0.63/getting-started.md

```diff
@@ -72,7 +72,12 @@ This sends telemetry to Microsoft by default. You can prevent the telemetry from
 
 - Using Visual Studio
 
-  - Open the solution file in the application folder in Visual Studio (e.g., `AwesomeProject/windows/AwesomeProject.sln` if you used `AwesomeProject` as `<projectName>`)
+  - Open the `windows` folder in the project directory (e.g., `AwesomeProject/windows` if you used `AwesomeProject` as `<projectName>`)
+  - In the `windows` folder, run:
```

Comment:
As above, the command needs to be run from the root of the repo (i.e. `AwesomeProject`) not the `windows` folder.